README.md

```diff
@@ -13,13 +13,17 @@ ember install ember-frost-test
 ## Testing Tools
 We are using the following tools:
 
-* [ember-cli-mocha](https://github.com/ember-cli/ember-cli-mocha) - This is our testing framework.
-It also includes, chaijs our assertion library.
+* [ember-cli-mocha](https://github.com/ember-cli/ember-cli-mocha) - This installs the [Mocha](http://mochajs.org)
+ testing framework.
+* [ember-cli-chai](https://github.com/ember-cli/ember-cli-chai) - This install the [Chai](http://chaijs.com)
+ assertion library.
 * [ember-hook](https://github.com/Ticketfly/ember-hook) - This is a tool we use to create a separation between the
 DOM and our items under test.
 * [ember-sinon](https://github.com/csantero/ember-sinon) - This is our method spying/stubbing/mocking tool.
```

Comment:
Use same format as for `ember-cli-mocha` and `ember-cli-chai`.  There is no link to Sinon